addons/apollo/functions/fnc_vehicleLoad.sqf

```diff
@@ -47,7 +47,7 @@ if (_retrieveVehicles == "ready") then {
 
     // Fix possible issue where physics don't activate until doing it manually (eg. shooting the object)
     _vehList params ["_vehFixPhysics"];
```

Comment:
Should probably also use proper array copy above that (`+` operator), where array should be copied for safety.